modules/ROOT/pages/apikit-odatav4-expand-feature.adoc

```diff
@@ -0,0 +1,98 @@
+= Implement the $expand Feature
+
+The `$expand` system query option reduces the amount of incoming network traffic and makes APIs easier to consume. Because consumers request only inline representations of the related entities of query results, the returned information avoids multiple network roundtrips. This benefits metered and latency-sensitive consumers.
+
+From APIkit for OData 1.3.0 onwards, this module supports automatic routing for ease of implementing the `$expand` system query option in the most common cases. Although this feature requires more preparation and configuration, it enables the reuse of already implemented flows when processing the `$expand` requests.
```

Comment:
"when processing the $expand requests" sounds weird to me (I'm not an english native so idk)

$expand requests are not inherently different for other requests. `$expand` is an option that can be added to any given request to ask for special processing of the indicated relationships.

$expand can be used in conjunction with almost any other system query option ($filter, $skip, $top, etc)

Reply:
It does not seem weird to me or to the editor, @iglosiggio.
Would it be more accurate to say "Although this feature requires more preparation and configuration, it enables the reuse of already implemented flows when including the `$expand` requests."?

Reply:
Lets keep it as-is. :)